### ISSUE TITLE
sumatrapdf hangs (working with sublime text)

### ISSUE THREAD

**Issue by amiroshn15** [sumatrapdfreader/sumatrapdf#703]
Hello, 
when I work in sublime text (in Windows 7), after a tex-file is built when switching to SumatraPDF it hangs (not every time - every other time). It happens often with any pdf file. I followed the instructions provided here https://github.com/sumatrapdfreader/sumatrapdf/wiki/Debugging-Sumatra and when it hanged got a WinDbg report. It is below:

```
0:004> .sympath+ SRV*c:\symbols*http://msdl.microsoft.com/download/symbols
Symbol search path is: srv*;SRV*c:\symbols*http://msdl.microsoft.com/download/symbols
Expanded Symbol search path is: cache*;SRV*https://msdl.microsoft.com/download/symbols;srv*c:\symbols*http://msdl.microsoft.com/download/symbols

************* Symbol Path validation summary **************
Response                         Time (ms)     Location
Deferred                                       srv*
Deferred                                       SRV*c:\symbols*http://msdl.microsoft.com/download/symbols
0:004> ~*kb

   0  Id: 78c.284 Suspend: 1 Teb: 000007ff`fffdd000 Unfrozen
 # RetAddr           : Args to Child                                                           : Call Site
00 000007fe`fd851b3b : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!ZwWriteFile+0xa
01 00000000`77991fd6 : 00000000`00000000 00000000`00000000 00000000`03dfbd10 00000000`0028d610 : KERNELBASE!WriteFile+0x77
*** ERROR: Symbol file could not be found.  Defaulted to export symbols for C:\Program Files\SumatraPDF\libmupdf.dll - 
02 000007fe`eba0d558 : 00000000`00000002 000007fe`ffce0369 00000000`037ae086 00000000`00000002 : kernel32!WriteFileImplementation+0x36
03 000007fe`eba0db31 : 00000000`00000000 000007fe`eb9fefd5 00000000`0028eab0 00000000`0028ebc8 : libmupdf!WebPGetInfo+0xae188
04 000007fe`eba0d8e0 : 00000000`00000000 00000000`00000146 00000000`0028eba0 00000000`00000002 : libmupdf!WebPGetInfo+0xae761
05 000007fe`eba0a4ac : 00000000`00000002 00000000`00000000 00000000`00000000 00000000`0028ecb8 : libmupdf!WebPGetInfo+0xae510
06 000007fe`eba0a60c : 00000000`00351965 000007fe`ebd477f0 000007fe`ebd477f0 00000000`00000002 : libmupdf!WebPGetInfo+0xab0dc
07 000007fe`eb9ff140 : 00000000`0028ed48 00000000`0028f188 00000000`0028ec90 00000000`0028ec90 : libmupdf!WebPGetInfo+0xab23c
08 000007fe`eb9fefc6 : 00000000`0028ed20 00000000`0028ed48 00000000`fffffff6 000007fe`eba00e78 : libmupdf!WebPGetInfo+0x9fd70
09 000007fe`eb9fe56c : 00000000`0028ed20 00000000`0028ed20 00000000`fffffff6 000007fe`eb9fde6e : libmupdf!WebPGetInfo+0x9fbf6
0a 000007fe`eb9fdb06 : 00000000`00000000 00000000`0028ed20 000007fe`ebae30a0 00000000`0028f200 : libmupdf!WebPGetInfo+0x9f19c
0b 000007fe`eb9fd8c1 : 00000000`0028f228 00000000`0028f250 00000000`00000100 000007fe`eba007f9 : libmupdf!WebPGetInfo+0x9e736
0c 000007fe`eb9fd416 : 000007fe`ebcf4d8c 00000000`0028f218 00000000`0028f2c0 00000000`00000000 : libmupdf!WebPGetInfo+0x9e4f1
0d 000007fe`eb9ff2f9 : 000007fe`ebd477f0 00000000`02848cc0 00000000`03e4f660 00000000`0028f218 : libmupdf!WebPGetInfo+0x9e046
0e 000007fe`eb81a64c : 000007fe`ebd477f0 00000000`00000000 00000000`00000004 000007fe`ebcefb90 : libmupdf!WebPGetInfo+0x9ff29
0f 000007fe`eb82a739 : 00000000`0039e870 000007fe`ebcefb90 000007fe`ebcf4d8c 00000000`00000915 : libmupdf!fz_set_aa_level+0x1dfc
10 000007fe`eb87ecb7 : 00000000`0028f3f9 00000000`03e4f660 00000000`0028f618 000007fe`ebcf58b0 : libmupdf!fz_rethrow_message_imp+0xe9
11 000007fe`eb8afba9 : 00000000`02071f90 000007fe`eb829e06 00000000`00000000 00000000`00001004 : libmupdf!pdf_open_document_no_run_with_stream+0xb7
*** ERROR: Module load completed but symbols could not be loaded for C:\Program Files\SumatraPDF\SumatraPDF.exe
12 00000001`3f8886b2 : 00000000`02071f90 00000000`02071f90 00000000`03dcfa90 00000000`0028f4a0 : libmupdf!pdf_open_document_with_stream+0x9
13 00000001`3f888263 : 00000000`02071f90 00000000`03e4f510 00000000`03d59700 00000000`03d59700 : SumatraPDF+0xa86b2
14 00000001`3f8656c4 : 00000000`03d59700 00000000`00396620 00000000`0028f618 00000000`03d59700 : SumatraPDF+0xa8263
15 00000001`3f7e3be7 : 00000000`00000000 00000000`0028f648 00000000`00000000 00000000`00000000 : SumatraPDF+0x856c4
16 00000001`3f7e4dd7 : 00000000`0038e870 00000000`00000001 00000000`00382f10 00000000`00000001 : SumatraPDF+0x3be7
17 00000001`3f809afe : 00000000`000506f4 00000000`0037fca0 00000000`00000005 00000000`00000113 : SumatraPDF+0x4dd7
18 00000000`77889c11 : 00000000`00000000 00000000`0028f940 00000000`00000001 00000000`00000000 : SumatraPDF+0x29afe
19 00000000`7788992a : 00000000`0028f800 00000001`3f809950 00000000`0037fca0 00000000`00a05e20 : USER32!UserCallWinProcCheckWow+0x1ad
1a 00000001`3f7f1aeb : 00000000`00000001 00000000`0028f940 00000000`00000001 00000001`3f809950 : USER32!DispatchMessageWorker+0x3d0
1b 00000001`3f7f24ad : 00000000`00000001 00000000`0028f940 00000000`00362a70 00000000`00000000 : SumatraPDF+0x11aeb
1c 00000001`3f8e11e7 : 00000000`00000000 00000000`00000001 00000000`00000000 00000000`00000000 : SumatraPDF+0x124ad
1d 00000000`77985a4d : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : SumatraPDF+0x1011e7
1e 00000000`77abb831 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : kernel32!BaseThreadInitThunk+0xd
1f 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!RtlUserThreadStart+0x1d

   1  Id: 78c.428 Suspend: 1 Teb: 000007ff`fffdb000 Unfrozen
 # RetAddr           : Args to Child                                                           : Call Site
00 000007fe`fd8510dc : 00000000`03cc2100 00000000`77991a7a 00000000`003bf600 00000000`00000000 : ntdll!ZwWaitForSingleObject+0xa
01 00000001`3f80eb8a : 00000000`00000000 00000000`01fff7d9 00000000`00000000 00000000`000000d8 : KERNELBASE!WaitForSingleObjectEx+0x79
02 00000000`77985a4d : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : SumatraPDF+0x2eb8a
03 00000000`77abb831 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : kernel32!BaseThreadInitThunk+0xd
04 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!RtlUserThreadStart+0x1d

   2  Id: 78c.ef4 Suspend: 1 Teb: 000007ff`fffd9000 Unfrozen
 # RetAddr           : Args to Child                                                           : Call Site
00 000007fe`fd8510dc : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!ZwWaitForSingleObject+0xa
01 00000001`3f827344 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`000000f4 : KERNELBASE!WaitForSingleObjectEx+0x79
02 00000000`77985a4d : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : SumatraPDF+0x47344
03 00000000`77abb831 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : kernel32!BaseThreadInitThunk+0xd
04 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!RtlUserThreadStart+0x1d

   3  Id: 78c.b54 Suspend: 1 Teb: 000007ff`fffd7000 Unfrozen
 # RetAddr           : Args to Child                                                           : Call Site
00 000007fe`fd851430 : 00000000`00000001 00000000`00000001 00000000`00000000 00000000`00000000 : ntdll!NtWaitForMultipleObjects+0xa
01 00000000`77991753 : 00000000`04b4f8b8 00000000`04b4f8b0 00000000`00000000 00000000`00000000 : KERNELBASE!WaitForMultipleObjectsEx+0xe8
02 00000001`3f844aed : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : kernel32!WaitForMultipleObjectsExImplementation+0xb3
03 00000000`77985a4d : 00000000`00000000 00000000`00000144 00000000`00000000 00000000`00000000 : SumatraPDF+0x64aed
04 00000000`77abb831 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : kernel32!BaseThreadInitThunk+0xd
05 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!RtlUserThreadStart+0x1d

#  4  Id: 78c.ad4 Suspend: 1 Teb: 000007ff`fffd5000 Unfrozen
 # RetAddr           : Args to Child                                                           : Call Site
00 00000000`77b840b8 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!DbgBreakPoint
01 00000000`77985a4d : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!DbgUiRemoteBreakin+0x38
02 00000000`77abb831 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : kernel32!BaseThreadInitThunk+0xd
03 00000000`00000000 : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!RtlUserThreadStart+0x1d
0:004> lmf
start             end                 module name
00000000`77870000 00000000`7796a000   USER32   C:\Windows\system32\USER32.dll
00000000`77970000 00000000`77a90000   kernel32 C:\Windows\system32\kernel32.dll
00000000`77a90000 00000000`77c39000   ntdll    C:\Windows\SYSTEM32\ntdll.dll
00000000`77c50000 00000000`77c53000   normaliz C:\Windows\system32\normaliz.DLL
00000000`77c60000 00000000`77c67000   PSAPI    C:\Windows\system32\PSAPI.DLL
00000001`3f7e0000 00000001`3f9f1000   SumatraPDF C:\Program Files\SumatraPDF\SumatraPDF.exe
000007fe`eb720000 000007fe`eb736000   sendmail C:\Windows\System32\sendmail.dll
000007fe`eb740000 000007fe`eb7fa000   uiautomationcore C:\Windows\system32\uiautomationcore.dll
000007fe`eb800000 000007fe`ebd98000   libmupdf C:\Program Files\SumatraPDF\libmupdf.dll
000007fe`ec4c0000 000007fe`ec514000   OLEACC   C:\Windows\system32\OLEACC.dll
000007fe`f28f0000 000007fe`f2a15000   dbghelp  C:\Windows\system32\dbghelp.dll
000007fe`f5d90000 000007fe`f5e01000   WINSPOOL C:\Windows\system32\WINSPOOL.DRV
000007fe`f7f70000 000007fe`f7f73000   api_ms_win_core_synch_l1_2_0 C:\Windows\system32\api-ms-win-core-synch-l1-2-0.DLL
000007fe`fa7b0000 000007fe`fa7ba000   ktmw32   C:\Windows\system32\ktmw32.dll
000007fe`fa960000 000007fe`fa967000   MSIMG32  C:\Windows\system32\MSIMG32.dll
000007fe`fb770000 000007fe`fb79d000   ntmarta  C:\Windows\system32\ntmarta.dll
000007fe`fb7a0000 000007fe`fb901000   WindowsCodecs C:\Windows\system32\WindowsCodecs.dll
000007fe`fb950000 000007fe`fb968000   dwmapi   C:\Windows\system32\dwmapi.dll
000007fe`fbb10000 000007fe`fbd26000   gdiplus  C:\Windows\WinSxS\amd64_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.7601.19061_none_2b299db671e86e03\gdiplus.dll
000007fe`fbd30000 000007fe`fbd86000   uxtheme  C:\Windows\system32\uxtheme.dll
000007fe`fbdb0000 000007fe`fbfa4000   COMCTL32 C:\Windows\WinSxS\amd64_microsoft.windows.common-controls_6595b64144ccf1df_6.0.7601.18837_none_fa3b1e3d17594757\COMCTL32.dll
000007fe`fc4b0000 000007fe`fc5dc000   propsys  C:\Windows\system32\propsys.dll
000007fe`fc8d0000 000007fe`fc8dc000   version  C:\Windows\system32\version.DLL
000007fe`fd670000 000007fe`fd6c7000   apphelp  C:\Windows\system32\apphelp.dll
000007fe`fd6d0000 000007fe`fd6df000   CRYPTBASE C:\Windows\system32\CRYPTBASE.dll
000007fe`fd840000 000007fe`fd84f000   profapi  C:\Windows\system32\profapi.dll
000007fe`fd850000 000007fe`fd8bc000   KERNELBASE C:\Windows\system32\KERNELBASE.dll
000007fe`fd8c0000 000007fe`fd8da000   DEVOBJ   C:\Windows\system32\DEVOBJ.dll
000007fe`fd8e0000 000007fe`fd8e4000   api_ms_win_downlevel_version_l1_1_0 C:\Windows\system32\api-ms-win-downlevel-version-l1-1-0.dll
000007fe`fd9d0000 000007fe`fd9d4000   api_ms_win_downlevel_shlwapi_l1_1_0 C:\Windows\system32\api-ms-win-downlevel-shlwapi-l1-1-0.dll
000007fe`fdb50000 000007fe`fdb54000   api_ms_win_downlevel_user32_l1_1_0 C:\Windows\system32\api-ms-win-downlevel-user32-l1-1-0.dll
000007fe`fdb60000 000007fe`fdb96000   CFGMGR32 C:\Windows\system32\CFGMGR32.dll
000007fe`fdba0000 000007fe`fdba5000   api_ms_win_downlevel_advapi32_l1_1_0 C:\Windows\system32\api-ms-win-downlevel-advapi32-l1-1-0.dll
000007fe`fdbb0000 000007fe`fdbb3000   api_ms_win_downlevel_normaliz_l1_1_0 C:\Windows\system32\api-ms-win-downlevel-normaliz-l1-1-0.dll
000007fe`fdbc0000 000007fe`fdbc4000   api_ms_win_downlevel_ole32_l1_1_0 C:\Windows\system32\api-ms-win-downlevel-ole32-l1-1-0.dll
000007fe`fdbd0000 000007fe`fdbee000   USERENV  C:\Windows\system32\USERENV.dll
000007fe`fdc70000 000007fe`fdd09000   CLBCatQ  C:\Windows\system32\CLBCatQ.DLL
000007fe`fdd10000 000007fe`fddeb000   advapi32 C:\Windows\system32\advapi32.DLL
000007fe`fddf0000 000007fe`fddfe000   LPK      C:\Windows\system32\LPK.dll
000007fe`fde00000 000007fe`fde67000   GDI32    C:\Windows\system32\GDI32.dll
000007fe`fde80000 000007fe`fe148000   iertutil C:\Windows\system32\iertutil.dll
000007fe`fe150000 000007fe`fe2d5000   urlmon   C:\Windows\system32\urlmon.dll
000007fe`fe2e0000 000007fe`fe3b7000   OLEAUT32 C:\Windows\system32\OLEAUT32.dll
000007fe`fe3c0000 000007fe`fe431000   SHLWAPI  C:\Windows\system32\SHLWAPI.dll
000007fe`fe440000 000007fe`ff1c9000   SHELL32  C:\Windows\system32\SHELL32.dll
000007fe`ff1d0000 000007fe`ff3d3000   ole32    C:\Windows\system32\ole32.dll
000007fe`ff3e0000 000007fe`ff40e000   IMM32    C:\Windows\system32\IMM32.DLL
000007fe`ff410000 000007fe`ff679000   WININET  C:\Windows\system32\WININET.dll
000007fe`ff680000 000007fe`ff69f000   sechost  C:\Windows\SYSTEM32\sechost.dll
000007fe`ff6f0000 000007fe`ff742000   WLDAP32  C:\Windows\system32\WLDAP32.dll
000007fe`ff750000 000007fe`ff859000   MSCTF    C:\Windows\system32\MSCTF.dll
000007fe`ff860000 000007fe`ffa37000   SETUPAPI C:\Windows\system32\SETUPAPI.dll
000007fe`ffa40000 000007fe`ffad7000   COMDLG32 C:\Windows\system32\COMDLG32.dll
000007fe`ffae0000 000007fe`ffb7f000   msvcrt   C:\Windows\system32\msvcrt.dll
000007fe`ffba0000 000007fe`ffccd000   RPCRT4   C:\Windows\system32\RPCRT4.dll
000007fe`ffcd0000 000007fe`ffd9a000   USP10    C:\Windows\system32\USP10.dll
```

**Comment by GitHubRulesOK** [comment]
@kjk 
TLDR; old historic report not actioned at that time, closure ?
@amiroshn15 
Appologies this appears to have been overlooked for so long, If it is still causing you occasional problems please test with a recent build, https://www.sumatrapdfreader.org/prerelease.html and re-report

**Comment by kjk** [comment]
Yes. Plus the callstack has invalid symbols and doesn't help. Plus there's been many changes since.

Please try with the latest version and if still happens, a new callstack would be necessary.
